src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable-line prefer-template */
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] Hi, I would advise you not to disable Linter because Linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Also, the general requirements prohibit disabling it.
- Note: Check if your app is working accordingly(Adding and Removing books ) before requesting for re-review, this will help you to save time.